applications/FSIApplication/python_scripts/fsi_analysis.py

```diff
@@ -111,7 +111,8 @@ def _GetOrderOfProcessesInitialization(self):
                 "fluid_gravity",
                 "fluid_initial_conditions_process_list",
                 "fluid_boundary_conditions_process_list",
-                "fluid_auxiliary_process_list"]
```

Comment:
@roigcarlo I did this commit in order to do the transition and avoid a behaviour change